znh5md/format/__init__.py

```diff
@@ -67,6 +67,25 @@ def decode_boundary(value) -> np.ndarray:
 OBSERVABLES_GRP = [GRP.energy, GRP.stress]
 
 
+@dataclasses.dataclass
+class StepTimeChunkUnits:
+    value: str
+    time: str
+
+
+# We store everything in ASE units
+UNITS_PER_GRP = {
+    GRP.position: StepTimeChunkUnits(value="Angstrom", time="fs"),
+    GRP.velocity: StepTimeChunkUnits(value="Angstrom/fs", time="fs"),
+    GRP.forces: StepTimeChunkUnits(value="eV/Angstrom", time="fs"),
+    GRP.energy: StepTimeChunkUnits(value="eV", time="fs"),
+    GRP.stress: StepTimeChunkUnits(value="GPa", time="fs"),
+    GRP.momentum: StepTimeChunkUnits(value="eV/fs", time="fs"),
+    GRP.edges: StepTimeChunkUnits(value="Angstrom", time="fs"),
+    GRP.boundary: StepTimeChunkUnits(value="Angstrom", time="fs"),
+}
```

Comment:
ah so these are like metadata in the H5 file?

Reply:
Yes, these are the ASE units. This is a bit inflexible right now, such that if you don't go through ASE there is no nice way of defining units.